tests/config/get-plugins.js

```diff
@@ -0,0 +1,39 @@
+import { createRequire } from "node:module";
+import path from "node:path";
+import createEsmUtils from "esm-utils";
+import { satisfies } from "semver";
+import getPrettier from "./get-prettier.js";
+
+const { __dirname } = createEsmUtils(import.meta);
+
+async function getPluginsInternal() {
+  if (!process.env.TEST_STANDALONE) {
+    return [path.join(__dirname, "../../src/index.ts")];
+  }
+
+  const require = createRequire(import.meta.url);
+  const prettier = await getPrettier();
+
+  const prettierPath = path.dirname(require.resolve("prettier"));
+  const pluginPrefix = satisfies(prettier.version, "^2.3.0")
```

Comment:
If this PR is explicitly dropping support for v2, I wonder if this should be kept, specifying `3.0.0` instead?